FirefoxBezpieczenstwo.md

```diff
@@ -57,6 +57,8 @@ Wyłączyć można je w ustawieniach Firefoxa w zakładce **Prywatność i Bezpi
 
 ![S](https://user-images.githubusercontent.com/41945903/68146588-a0a61080-ff38-11e9-882d-c97b505ef4e6.png)
 
+dodatkowo, w **about:config** wpisz **telemetry** i ustaw wszystko na **false**
```

Comment:
Jest to zbyt ogólne stwierdzenie. W różnych wersjach Firefoxa mogą być tam powpisywane najróżniejsze ustawienia.